plot/index.html

```diff
@@ -0,0 +1,69 @@
+<html id="plot">
+
+<head>
+  <script src='../../chaise/chaise-config.js'></script>
+  <script src='../../chaise/scripts/vendor/jquery-1.11.1.min.js'></script>
+  <script src='../../chaise/scripts/vendor/bootstrap-3.3.7.min.js'></script>
+  <script src='vendor/plotly-latest.min.js'></script>
+  <script src='vendor/moment.min.js'></script>
+  <script src='../../chaise/scripts/vendor/angular.js'></script>
+  <script src='../../chaise/scripts/vendor/angular-sanitize.js'></script>
+  <script src='../../chaise/common/vendor/angular-cookies.min.js'></script>
+  <link rel="stylesheet" type="text/css" href="../../chaise/styles/vendor/bootstrap.min.css">
+  <link rel="stylesheet" type="text/css" href="../../chaise/common/styles/app.css">
+  <link rel="stylesheet" type="text/css" href="../../chaise/common/styles/appheader.css">
+  <script src='../../chaise/common/alerts.js'></script>
+  <script src='../../chaise/common/authen.js'></script>
+  <script src='../../chaise/common/storage.js'></script>
+  <script src='../../chaise/common/errors.js'></script>
+  <script src='../../chaise/common/filters.js'></script>
+  <script src='../../chaise/common/inputs.js'></script>
+  <script src='../../chaise/common/modal.js'></script>
+  <script src='../../chaise/common/login.js'></script>
+  <script src='../../chaise/common/navbar.js'></script>
+  <script src='../../chaise/common/utils.js'></script>
+  <script src='../../chaise/common/validators.js'></script>
+  <script src='../../chaise/common/config.js'></script>
+
+  <script src='../../chaise/scripts/vendor/ui-bootstrap-tpls-2.5.0.min.js'></script>
+  <script src='../../ermrestjs/ermrest.js'></script>
+  <script src="plot-config.js"></script>
+  <script src="plot.app.js"></script>
+  <link rel="stylesheet" type="text/css" href="styles/plot.css">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+
+<body class="chaise-body">
+  <div class="configure-container" ng-app="chaise.configure-plotApp"></div>
+  <div class="mainContainer">
```

Comment:
To be consistent with the rest of the chaise apps, this class should be `app-container`. 

We use `app-container` to denote the element that wraps the whole app (including the navbar while excluding the config app).